cachito/common/packages_data.py

```diff
@@ -168,6 +168,7 @@ def load(self, file_name: Union[str, Path]) -> None:
 
         with open(file_name, "r", encoding="utf-8") as f:
             data = json.load(f)
+            log.info("Loaded file %s: %s", file_name, data)
```

Comment:
This will often be a huge log message and will appear in the /logs for a request. Not sure if we can live without it though.

Reply:
I'm desperate. :sweat_smile: 
We could mark this to be removed once we believe everything is stable.

Reply:
Sounds good to me